packages/wordcount/src/index.js

```diff
@@ -37,7 +48,7 @@ function loadSettings( type, userSettings ) {
 		);
 	}
 
-	settings.type = type || settings.l10n.type;
+	settings.type = type || settings.l10n?.type;
```

Comment:
`type` is not an optional argument, it should be one of the valid strings. I suspect we can simplify things here.

Reply:
Oh, yeah, thanks.Fixed in aa335aa.